olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CatalogVersionRequest.h

```diff
@@ -32,22 +33,19 @@ namespace dataservice {
 namespace read {
 
 /**
- * @brief The CatalogVersionRequest class encapsulates the fields required to request
- * the Catalog configuration.
+ * @brief The CatalogVersionRequest class encapsulates the fields required to
+ * request the Catalog configuration.
  */
 class DATASERVICE_READ_API CatalogVersionRequest final {
  public:
-
   /**
    * @brief Mandatory for versioned layers; the beginning of the range of
    * versions you want to get (exclusive). By convention -1 indicates the
    * initial version before the first publication. After the first publication,
    * the catalog version is 0.
    * @return the start version.
    */
-  inline int64_t GetStartVersion() const {
-    return start_version_;
-  }
+  inline int64_t GetStartVersion() const { return start_version_; }
```

Comment:
Are you sure this formatting is correct? For me, using the clang-format, it never formats to one-liners.

Reply:
yes, that's clang-format's outcome